package.json

```diff
@@ -68,6 +68,9 @@
     "try-require": "^1.0.0",
     "update-notifier": "^2.5.0"
   },
+  "optionalDependencies": {
```

Comment:
We'll need a version bump of Superstatic too; what are thinking? By the semantic versioning rules this...shouldn't break backwards compatibility by itself, but if we do the node 8 bump it definitely will.

Reply:
I'm not opposed to doing a major version bump if we want to drop node 6, but I would do that outside this particular PR (it'd be part of the release process). Removing node 6 testing should be a different PR if it's not required to drop it in this PR.

Reply:
I've made a change that will require dropping node 6 and below.